release/make_CARLOS_deb.sh

```
@@ -385,45 +385,25 @@ chmod 711 ${RELEASE_DIR}/${DEBNAME}/var/lib/${PACKAGE}/restore.sh
+FLYWAY_VERSION=11.14.0
+SCHEMA_OUT=${RELEASE_DIR}/${DEBNAME}/var/lib/${PACKAGE}/schema
+echo "bundling Flyway migration set from database/mysql/migration/"
+mkdir -p ${SCHEMA_OUT}
+cp -R ./database/mysql/migration ${SCHEMA_OUT}/migration
+# expire_openodoc.sql is a dev-only credential-expiry helper kept for existing tooling.
+cp ./database/mysql/expire_openodoc.sql ${SCHEMA_OUT}/ 2>/dev/null || true
+
+echo "bundling Flyway ${FLYWAY_VERSION} CLI (offline-capable, self-contained JRE)"
+# The bundled linux CLI tarball is on Red Gate's mirror; Maven Central only ships the jar.
+curl -fsSL "https://download.red-gate.com/maven/release/com/redgate/flyway/flyway-commandline/${FLYWAY_VERSION}/flyway-commandline-${FLYWAY_VERSION}-linux-x64.tar.gz" \
+  | tar xz -C ${SCHEMA_OUT}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)